tests/settings/SettingTest.py

```diff
@@ -14,9 +14,12 @@ class SettingTest(unittest.TestCase):
     def test_construction(self):
         self.assertRaises(ValueError, Setting, '', 2, 2)
         self.assertRaises(TypeError, Setting, '', '', '', from_cli=5)
+        self.assertRaisesRegex(TypeError, 'to_append.*',
+                               Setting, '', '', '', to_append=10)
 
     def test_path(self):
-        self.uut = Setting('key', ' 22\n', '.' + os.path.sep, True)
+        self.uut = Setting(
+            'key', ' 22\n', '.' + os.path.sep, strip_whitespaces=True)
```

Comment:
These `strip_whitespaces=` are not necessary any longer.

I have no objection to them being changed, as it is more readable, but it isnt a required change for this changeset.